packages/deployer/config/config.json.example

```diff
@@ -9,21 +9,15 @@
         "AirnodeRrp": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
       },
       "id": "31337",
-      "providerNames": [
-        "EVM local"
-      ],
+      "providers": {
+        "local": {
+          "url": "${LOCAL_PROVIDER_URL}"
+        }
+      },
       "type": "evm"
     }
   ],
   "environment": {
-    "chainProviders": [
```

Comment:
🎉

Reply:
This just moved directly to `providers` right?